<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NumericKeyboard</class>
 <widget class="QWidget" name="NumericKeyboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget #NumericKeyboard{
background-color: #FFFFFF;
}

</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>360</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{
border: 2px solid #Fb8800;
background-color: #FFFFFF;
}

QPushButton{
border: 2px solid #Fb8800;
font-family: &quot;Microsoft YaHei&quot;;
font-weight:bold;
font-size:15px;
color: #303030;
}

QPushButton:disabled{
background-color: rgba(204,204,204,50)
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="btnDelete">
    <property name="geometry">
     <rect>
      <x>266</x>
      <y>70</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnDelete{
background-color: #B3B3B3;
background-image: url(:/icons/btnDelete.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnHEX">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>178</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:#B3B3B3;
}
QPushButton:hover{
background-color:#3399CC;
}
QPushButton:checked{
background-color:#3399CC;
}</string>
    </property>
    <property name="text">
     <string>HEX</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="btnBIN">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>284</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:#B3B3B3;
}
QPushButton:hover{
background-color:#3399CC;
}
QPushButton:checked{
background-color:#3399CC;
}</string>
    </property>
    <property name="text">
     <string>BIN</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="btnSwitch">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>284</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnSwitch{
background-color: #B3B3B3;
background-image: url(:/icons/btnSwitch.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn4">
    <property name="geometry">
     <rect>
      <x>138</x>
      <y>178</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed {
    /* 改变背景色 */
    background-color:rgb(253,196,128);
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="btnC">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>178</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed {
    /* 改变背景色 */
    background-color:rgb(253,196,128);
}</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="btnD">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>178</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed {
    /* 改变背景色 */
    background-color:rgb(253,196,128);
}</string>
    </property>
    <property name="text">
     <string>D</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="btnDEC">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>230</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:#B3B3B3;
}
QPushButton:hover{
background-color:#3399CC;
}
QPushButton:checked{
background-color:#3399CC;
}</string>
    </property>
    <property name="text">
     <string>DEC</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="btnCancel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnCancel{
background-color: #FF0000;
color:#FFFFFF;
}</string>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn5">
    <property name="geometry">
     <rect>
      <x>202</x>
      <y>178</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed {
    /* 改变背景色 */
    background-color:rgb(253,196,128);
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="btnF">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>230</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed {
    /* 改变背景色 */
    background-color:rgb(253,196,128);
}</string>
    </property>
    <property name="text">
     <string>F</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="btn1">
    <property name="geometry">
     <rect>
      <x>138</x>
      <y>230</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed {
    /* 改变背景色 */
    background-color:rgb(253,196,128);
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPoint">
    <property name="geometry">
     <rect>
      <x>266</x>
      <y>284</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed {
    /* 改变背景色 */
    background-color:rgb(253,196,128);
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="btnClear">
    <property name="geometry">
     <rect>
      <x>202</x>
      <y>70</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnClear{
background-color: #B3B3B3;
}</string>
    </property>
    <property name="text">
     <string>clr</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn6">
    <property name="geometry">
     <rect>
      <x>266</x>
      <y>178</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed {
    /* 改变背景色 */
    background-color:rgb(253,196,128);
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="btnA">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>124</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed {
    /* 改变背景色 */
    background-color:rgb(253,196,128);
}</string>
    </property>
    <property name="text">
     <string>A</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="btn2">
    <property name="geometry">
     <rect>
      <x>202</x>
      <y>230</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed {
    /* 改变背景色 */
    background-color:rgb(253,196,128);
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="btnOk">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>70</y>
      <width>60</width>
      <height>104</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnOk{
background-color: #00BD00;
color:#FFFFFF;
}
</string>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn9">
    <property name="geometry">
     <rect>
      <x>266</x>
      <y>124</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed {
    /* 改变背景色 */
    background-color:rgb(253,196,128);
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="btnLeft">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>70</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnLeft{
background-color: #B3B3B3;
}</string>
    </property>
    <property name="text">
     <string>&lt;&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRight">
    <property name="geometry">
     <rect>
      <x>138</x>
      <y>70</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnRight{
background-color: #B3B3B3;
}</string>
    </property>
    <property name="text">
     <string>&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn8">
    <property name="geometry">
     <rect>
      <x>202</x>
      <y>124</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed {
    /* 改变背景色 */
    background-color:rgb(253,196,128);
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="btn3">
    <property name="geometry">
     <rect>
      <x>266</x>
      <y>230</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed {
    /* 改变背景色 */
    background-color:rgb(253,196,128);
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="btn0">
    <property name="geometry">
     <rect>
      <x>138</x>
      <y>284</y>
      <width>124</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed {
    /* 改变背景色 */
    background-color:rgb(253,196,128);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn7">
    <property name="geometry">
     <rect>
      <x>138</x>
      <y>124</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed {
    /* 改变背景色 */
    background-color:rgb(253,196,128);
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="btnB">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>124</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed {
    /* 改变背景色 */
    background-color:rgb(253,196,128);
}</string>
    </property>
    <property name="text">
     <string>B</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="btnSign">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>284</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed {
    /* 改变背景色 */
    background-color:rgb(253,196,128);
}</string>
    </property>
    <property name="text">
     <string>±</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="btnE">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed {
    /* 改变背景色 */
    background-color:rgb(253,196,128);
}</string>
    </property>
    <property name="text">
     <string>E</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup2</string>
    </attribute>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>380</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup2"/>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
